content/en/docs/reference/command-line-tools-reference/feature-gates/DisableKubeletCloudCredentialProviders.md

```diff
@@ -17,7 +17,9 @@ stages:
   - stage: stable
     defaultValue: true
     fromVersion: "1.31"
+    toVersion: "1.33"
```

Comment:
```suggestion
    toVersion: "1.32"
```

Reply:
@carlory,  small nit: toversion for `DisableKubeletCloudCredentialProviders` should be 1.32 instead of 1.33.